framework/test/jest/unit/specs/modules/chain/dpos/apply.spec.js

```diff
@@ -434,6 +434,63 @@ describe('dpos.apply()', () => {
 			});
 		});
 
+		// Reference: https://github.com/LiskHQ/lisk-sdk/issues/2423
+		describe('When summarizing round return value which is greater than Number.MAX_SAFE_INTEGER ', () => {
+			beforeEach(async () => {
+				feePerDelegate = new BigNum(Number.MAX_SAFE_INTEGER.toString()).add(
+					randomInt(10, 1000),
```

Comment:
I think this shouldn't be randomize. in some cases, it will be the same as using basic number and in some case it will be different

Reply:
Just tried to have a number larger than `Number.MAX_SAFE_INTEGER`. And instead of a fixed number tried to randomize it. I personally experience having random fixtures may catch some issues which you miss with a single static value.

Reply:
I think the test should be always deterministic rather than hoping some random number can catch the bug.

Reply:
Yes tests are always deterministic, but using a known range of input is a good practice compared to a static value. That's what we are doing everywhere in the tests, e.g. look at any fixtures files. Also this `randomInt` is already used inside the same file, its not thing new I introduced here.